specification/paths/traffic/trafficRoute.yml

```diff
@@ -0,0 +1,42 @@
+servers:
+  - url: https://api.woosmap.com
+tags:
+  - Traffic API
+description: >
+  Get distance, duration and path (as a polyline) for a pair of origin and destination, based on the recommended route between those two points (including traffic).
+security:
+  - PublicApiKeyAuth: []
+    RefererHeader: []
+  - PrivateApiKeyAuth: []
+parameters:
+  - $ref: "../../parameters/distance/origin.yml"
+  - $ref: "../../parameters/distance/destination.yml"
+  - $ref: "../../parameters/traffic/routing.yml"
+  - $ref: "../../parameters/distance/language.yml"
+  - $ref: "../../parameters/distance/units.yml"
+  - $ref: "../../parameters/distance/alternatives.yml"
+  - $ref: "../../parameters/distance/waypoints.yml"
+  - $ref: "../../parameters/traffic/departure_time.yml"
+  - $ref: "../../parameters/traffic/arrival_time.yml"
```

Comment:
Should be mentioned that arrival-time and departure_time should not be given together.